.eslintrc

```diff
@@ -1,3 +1,3 @@
 {
-  "extends": "ta-webapp",
+  "extends": "ta-webapp"
```

Comment:
did this break something? As I thought we were leaving trailing commas in objects with ES6.